components/stacked-vertical-bar-chart/styles.module.css

```diff
@@ -31,6 +31,12 @@
   white-space: nowrap;
 }
 
+.caption-value {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
I have literally no idea how to make these ellipsis work. Don't you know any css trick?

![s](https://user-images.githubusercontent.com/12775440/82666687-af36e980-9c36-11ea-9fbf-010f4b336a21.png)

Reply:
I think you can avoid ellipsis here. You do not need it. Also, the right label may be redundant. Just add minimal width for the inner element to fit the text. It would be clear enough, I think.

I've been thinking about this last night, we should probably use `<progress>` and `<meter>` where it's possible.

Reply:
Didn't know about those elements. We should definitely use them. :)

Reply:
is it ok if caption won't be there at all? I guess there is no way to add caption inside of progress/meter

Reply:
Sorry, it [was not a good idea](https://stackoverflow.com/a/41430146/7205866). We should probably stick with some kind of chart.

I will take a look at this during next week if I have time.

Reply:
I will rewrite this to pure CSS and we will see how it looks